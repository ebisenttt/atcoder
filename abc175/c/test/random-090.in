23987348262226 188776562232469 128818475985914
