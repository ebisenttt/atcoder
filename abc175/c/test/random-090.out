104831127723688
